client/src/pages/Dashboard/Dashboard.tsx

```diff
@@ -34,15 +33,14 @@ export default function Dashboard(): JSX.Element {
 
   return (
     <Grid container component="main" className={`${classes.root} ${classes.dashboard}`} direction="row">
-    <NavBar />
-    <Grid item className={classes.drawerWrapper}>
-      <ChatSideBanner loggedInUser={loggedInUser} />
-    </Grid>
-    <Grid item>
-      <Board></Board>
-    </Grid>
+      <NavBar />
```

Comment:
Is this formatting intended?

Reply:
Are you referring to the indentation? I added them because the linter wouldn't allow the code to compile